src/Utility/CommandHelper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\update_helper\Utility;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+
+class CommandHelper implements LoggerAwareInterface {
```

Comment:
Maybe we could make a service from this? Then we could nicely inject `update_helper.updater` and also we could inject it in `ApplyConfigurationUpdateCommands` for Drush services.